src/Microsoft.Health.Client/Microsoft.Health.Client.csproj

```diff
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netcoreapp3.1</TargetFramework>
+        <TargetFrameworks>$(SupportedFrameworks);</TargetFrameworks>
     </PropertyGroup>
 
     <ItemGroup>
         <PackageReference Include="Ensure.That" Version="9.2.0" />
         <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.5.0" />
-        <PackageReference Include="Microsoft.Extensions.Options" Version="3.1.7" />
-        <PackageReference Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" Version="5.5.0" />    
+        <PackageReference Include="Microsoft.Extensions.Options" Version="$(SdkPackageVersion)" />
```

Comment:
is this always the case?

Reply:
There are a few other instances of this. Not sure if it was deliberate.

Reply:
Sorry just clarifying whats the case :P?
Most `Microsoft.Extensions.*` packages version together and are in sync with the runtime. I've added this variable to make that easier